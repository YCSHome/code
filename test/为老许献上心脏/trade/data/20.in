400 500 500 
385 741517521448328
53 274978971012594
222 698447981264061
1 955320927142606
46 825205384953074
353 692640378088002
219 887577068751942
327 956444382194208
231 905220080726404
287 842103792189207
42 422119513567252
46 643668993361591
297 153834965466417
257 812936497770381
100 76807921291066
118 210313416345354
306 589773866932169
257 599230265807126
141 800595528932718
73 236694124448359
316 129335476882676
210 967559193051985
229 182364400634164
391 882304901288010
225 413574947378451
268 949691814515376
174 347122994233921
227 213291848684389
240 722066201019703
239 130071676096275
132 532883205330333
42 402823384082994
273 233353719326795
266 662744681757043
144 489281120650659
385 886191874691948
389 303117646040836
398 148121585086969
242 716134600274516
352 475026724579334
307 14446407261782
113 772372608683142
136 354163512479702
246 26864036340220
127 397620791307084
268 882156249631901
322 6338716608238
398 708827172040928
125 759950241469117
122 595832109767244
149 283458672305045
257 502212075672377
363 304922346480945
263 783670708314561
352 113352865841285
5 22498222730465
56 469541053039435
296 591157501710098
69 453459684778303
373 28828241576286
268 313182289670252
33 206544573497554
270 251929872019631
243 804444342073321
360 597522420465381
81 69036843606302
252 798645710950766
384 876877775894606
214 857916949201731
117 470280174458742
297 671280153629176
142 371033348495877
210 452549092893172
180 709878970376706
99 97739108229869
126 823371466285100
96 901159907367982
351 438390640004006
278 296401653589452
248 826701791426892
393 543032658119947
240 331144467615794
18 17986480438283
122 693800911962717
144 581921257152262
320 807610192040466
11 161206302757601
137 583063294788757
66 107453453093144
384 465221619090541
364 803688005810345
158 864739327557177
341 367777121840634
212 225600876055950
386 987728363668593
225 288621438388949
80 936708667181533
200 297959826708890
18 254864334420606
323 317415420973600
217 224384628425879
231 61051704079991
116 271344690408838
342 729093651458067
144 523208802194785
102 334269456633998
352 543524918369647
189 629325147579607
40 754579642047260
74 702095350634728
106 804857439766269
283 242287559683815
376 589122902282521
352 326236356135865
163 81557700131996
356 522569919166823
131 639114979744592
258 976321904900720
360 974045934162522
42 189382152900545
253 166212867728002
328 372164113470215
122 864770412401290
82 448549301040912
317 451476913601535
337 831029473632462
163 753058374352347
245 935478543932465
276 465183796444720
233 624204630674340
74 430089208169238
229 121698608935016
36 856141630807582
19 550533214529989
70 575802184972476
9 600739586656286
163 140569152227863
4 111885776314728
343 685157785514489
318 322841969546046
57 925424158708481
282 849920940208647
315 864804757233280
183 533293670255605
373 642726729983996
328 829314873680739
54 970314428080359
365 797436671353059
224 473519100217742
14 608384811539223
134 902429424833313
127 505573702808304
225 919822233072564
121 574440787172515
158 451390885322925
55 192479577272685
319 164075958138887
389 525978905254516
42 878480999363565
252 68464690285700
303 866698655867231
338 734691464951256
1 707666248269605
326 348233247361674
154 748779507790862
353 775819392732013
286 759335907000846
390 968686754210640
293 153074131244341
242 392591749803652
153 614512762432050
170 49214133946824
59 307373762312156
143 308027351135710
84 750284170555244
151 913279569235041
352 507594093214749
389 137891747379972
256 573090856220410
173 761502811078638
313 962551409650640
280 339342902111318
243 586540613372411
13 689093243699153
292 113932584919321
11 543055507206879
326 262930692549462
234 443071635417790
60 232301314468382
212 25524476203295
141 705212557659331
18 882250274075455
101 385536259087453
339 562088610753819
111 730981296050885
227 421764038702383
113 458187449396099
274 445987004604702
4 381444955153468
87 561240578714132
82 163906754426061
57 476629506808288
343 356617955391511
3 484614704091636
274 509335920385812
283 680210720900191
82 2786922626972
192 866886791295590
170 209420706958616
271 937412093635299
276 735852686464477
311 496566765642672
2 179275924819207
263 670167399103512
29 644874205279006
365 205358918535966
331 930450817337814
43 96168687159455
205 913776271186026
330 553020113992703
252 525942796872952
14 141878369410270
280 764122927472698
196 926004936580909
364 923303458851316
134 105833670133362
139 46917191111965
11 861742153374769
151 841257257539069
307 502276650592727
332 530517751867348
393 795007498557663
221 153847898264776
358 908727047236352
163 896715219845363
259 318361827323252
132 579258595536761
354 233320486120607
258 942214201686671
107 512812359787692
255 431932655845931
33 276267829416236
342 46472797321205
140 590312623243033
367 993550472825918
83 240417127082286
377 141851636284116
218 879210229750198
14 555459367016627
243 451372311954144
164 733111757440709
282 932548668157975
273 312140337542956
269 156941984858772
264 495775778469664
162 832363758489667
270 79175314666315
372 340151195242266
177 621989261596690
18 212691931772109
116 366741706343561
115 7211557494897
222 845754608546042
142 954942230063073
292 989730168727039
345 852690443200590
10 410046966323353
297 302824845597887
353 42713313187214
91 980537805370470
55 944320029151113
134 606127012011910
250 284733538857892
240 274555892180917
232 41397097405072
119 632239075005958
243 755454791404614
298 699803883754557
147 756541227652290
125 220446639891097
225 944054889761352
312 599824889569048
106 140881637844282
278 417584768474317
177 840285271772941
47 245834038568971
83 488217756558670
378 519720955277332
45 552355170538180
78 767125620718256
215 341141174673612
245 309770102959800
135 115729356755084
291 465193631594545
17 146357954135095
359 585576425767657
62 859588196231591
256 525933879432711
266 436382207208560
288 287052951492383
130 690774721885349
8 600715764027793
9 205343264897002
268 822189609837070
117 285782686343636
169 591686737641141
357 216784825398163
32 731835782479942
367 138980264523187
16 715019311462048
249 933594215638099
311 861957538121495
256 407224618468894
159 688274337354708
190 508979492816431
357 199307709367706
244 665124165218798
207 296079818224974
287 354708204637549
157 196653438801820
244 445309002289430
4 162504316155443
399 523432889075169
281 638705517736783
224 234117789696043
374 151591045836657
255 732362784862311
233 287358612119698
267 586209163809276
179 956511614627446
267 956970927340076
99 896302046205034
38 150658459652298
138 666733615280164
265 30584207911598
203 290587225659382
397 650848028662609
87 370417484950714
176 615300025238143
236 514611713587997
143 366176766050111
45 222774922109179
159 61467581718526
59 189092858352653
6 786502288184029
358 860811439173878
131 619352493471159
262 693926686307042
144 52755783273365
225 595495539238335
376 586085559031614
74 428852295452935
380 116839411799542
28 921935208293034
325 817331222490489
35 420759135445357
103 258832874468713
80 942836649469477
105 821143362749174
200 69720853281000
16 6220327496253
363 239216877034441
385 793544610291561
165 165384597745834
318 431687046048818
241 811934576853591
189 67807302880564
276 664926081207416
198 13409507237124
133 839773346121072
39 527485396088358
81 574843771204781
59 766470822442221
332 86421275452497
37 86105599399297
228 385392571373177
382 872741538204829
270 744429704558771
348 852635413496138
195 544536842597286
191 445131617711136
254 596622960623464
154 53979819906366
195 882083187651871
126 310750510296651
361 559194409056893
220 624630776418918
175 965994689939528
364 260521411940324
115 245711944588669
102 911283087023021
89 648004810001471
102 479632619615075
389 499596002500957
298 612870786902680
400 414531122691090
38 356568701786287
151 79674604883191
263 537011637905243
368 583157715499706
73 406012909121892
67 785894334362079
324 717040656564550
65 780721545353864
106 278508654918081
108 610762620599306
252 965247456488477
219 880460329164031
113 513174837948246
246 124578616858375
28 424311540236595
97 841997732707410
373 63677904508278
38 602214015386957
171 740700557264656
160 133890175695262
226 275802830078870
1 604011387871586
42 306962637601262
309 881244727373268
345 133443755378427
106 667867805175642
327 42041978264999
108 546983002838672
28 772590153231498
68 6390938875844
370 398926590607860
30 690730148862272
135 959279201047808
301 147878563478813
46 406293803858521
351 442583735494273
95 455704840344454
332 677688860336712
37 970325103038670
205 491250844402971
362 701404973959973
176 559034496783512
283 174113825207771
28 649366808264774
23 685696344040721
244 62683228121212
310 127961263518337
241 386280678625371
217 227206289520750
174 841461045475883
132 723533253673083
224 379846569812238
52 839944754249616
134 507855727042731
75 806434878719957
281 311427105857368
299 249015556462024
380 587947657659701
334 107873683473705
77 792655086805497
235 168440124702020
248 430132614686576
280 4811967306206
268 401443334922678
78 945901112631077
59 272214355532968
59 514888567345988
242 453803491941463
256 247122482365597
268 581953371420103
322 41241691248290
136 572711483308241
77 831051293563371
46 942334861584570
6 998765779435631
21 59655406104437
42 122445703273206
378 360996054866230
28 928502560348740
161 401210670572577
376 666419878546592
72 161498405814744
285 79707187593103
27 522269147202182
110 763236081829919
244 303396632405893
400 920620105590751
58 805259727940410
248 100062587660585
57 738671710343865
122 231569500412385
367 581008399276257
113 675162188500923
356 771163992032311
84 808218660828813
365 219137338198200
146 541717382939627
236 915791916590956
17 753910151219721
335 32137865753740
184 715604925396963
210 533334055675721
347 560774672138566
394 743629192299585
310 644705008997594
397 588368005550453
26 682483463214642
83 239972500313357
386 243166064303780
238 373250389835580
231 420087978138773
342 691896348632709
104 224916265743428
253 704930813787660
241 31768059536732
327 760426636083420
213 63320792916228
4 190654861802685
9 248506850504396
202 102158611913578
274 669609650627249
330 143364009311806
387 103336825948733
244 324377765645248
188 221381537829213
84 418139647025207
198 952731685737218
375 752867968222598
137 904722723391435
376 769848449135936
287 66425765070269
400 353142100974525
137 668751680031139
16 886100501364327
194 876336302600767
113 525552816563144
206 385304307809469
118 149285079894855
92 671080995424852
91 491518594167965
137 832912351719291
234 712641405604032
318 367026947920335
20 737242663554901
10 931211681476675
74 622490552095798
340 521501074326549
304 370160125437863
397 486786366290950
314 367297864946122
206 355693528928529
365 785708384974875
171 471253620756873
70 588349354277151
248 701648376415502
285 222078387937347
243 723960606427991
337 651841233900541
67 557851620402877
353 9705554456702
349 133962772698686
387 447811893754544
62 102397033181195
310 798138832264592
276 21440184178899
283 593184081797428
260 749567280412674
169 265153898784845
310 713034820941406
115 915108648474919
57 596222016269519
22 633524478791612
89 427967059768088
144 368125252674775
180 107184506759421
301 447783879705183
239 44338116528324
216 233654852535585
156 103536734349096
184 236558142027694
385 572856333609421
77 826598335946405
353 878336411517513
271 361034270078447
50 131163399333785
263 952017889126205
297 622324819748419
197 733304183562744
233 204841481625140
387 506679947600213
215 528117881997602
235 242534891014250
250 57814087849267
393 774558577728032
136 173120734181022
135 258069462343014
234 8196864283328
244 278785782607649
49 652211444909050
41 2855678887214
73 232025621599025
317 528536177399712
315 506003987611485
333 4580870962327
221 707363172898173
335 493828436619513
362 589978422111832
104 152033541879730
19 609600115571507
156 344517188453786
129 482689890872733
310 141499971218997
150 707408074590229
187 603721519433904
369 490144912980389
7 718686065366653
10 460893396362749
189 599839086629385
265 559956270845022
383 168307087901160
237 49990635856316
30 28636825359439
269 678574830295924
97 509571653687246
154 624882532700301
326 416678284554667
262 533068907400127
44 31218943039557
216 131209501513526
283 210965097211190
206 3379515039256
195 975802740087164
145 633835112417456
74 795029583944433
371 991177715148419
231 7340230861546
330 304351330824418
388 51176864252484
313 884563116661603
286 116987026161818
319 638366716998394
242 266913736217232
101 718703495512933
127 913356350125392
117 312241117462141
216 94734398729701
26 379777340805532
186 578099811666807
252 233508646746926
32 373582661962101
169 421260601436323
75 750914856590975
61 52780490142525
106 949862986750805
150 697113076416995
114 66328955675544
288 531144667921323
201 621250476742210
38 509132879359728
82 444610770196397
51 816623726035103
35 305820984032675
318 2256223054474
12 663566938667005
50 172460594339657
48 964170685341512
185 211189978611968
384 664799457372878
328 327932560612742
42 495884196213824
221 820188559913111
348 18006275573595
358 910853251389706
361 278944919244476
259 843250062331295
216 218134215440479
288 921855518292563
392 404937364242758
297 347742293498969
373 703132848310213
61 879302473371543
269 70132382763055
246 318617247035479
226 277056619224658
296 272909626766465
255 681187665171689
66 123569966520516
79 721814081985208
158 903495942224715
223 231489567328535
332 994318589218904
199 341157095877091
215 356628057458996
325 724674381709560
144 695145817756302
27 722367733272935
399 328957893767465
240 274646300545999
392 566395506443589
32 128517564984884
55 866543567576938
110 833935683349182
327 513863174556555
16 172551160568025
140 142646969568132
177 611153174016336
247 996804408228996
351 32447714688051
198 136833974342097
132 378365589618443
30 504645259128333
226 818794009056720
319 340293023349207
357 41773330034854
1 390761578006375
168 855458615717939
182 330157392235967
218 103280917800209
276 402277746878078
289 844851021257136
85 604501871727553
77 410336095701586
275 925385470251102
166 91143285402927
88 39298524226285
51 744144774255661
233 84024134549045
258 80228384289452
176 276449458334052
194 32850739594655
92 578840406605292
45 944164354102407
167 417277909797568
130 90746379030045
272 792379069123998
367 620859345339383
316 712412647650877
52 690740948547145
161 187205139231366
363 722263508397525
351 238356241821158
133 480761241211950
184 967966546128344
254 187234403348580
14 559492816635690
367 77075429118975
338 149572231476807
357 706086686919152
313 71812162622555
324 422360044211835
232 743319737360880
83 310779672778153
69 896230308031247
335 924430400757577
366 505866893233404
301 433572661784345
53 371003144448208
344 481014096839869
214 590017862805004
190 988705875636761
190 570873978083953
178 964666464030991
228 835251681992627
105 867694209815535
63 419663144382083
129 241509347391149
32 84220321736196
4 316629674224356
157 279568052365768
303 12108386343022
85 191122636897361
146 606526522448567
335 610922147155211
258 236347275285853
240 713071274468667
280 648707135989060
150 475690769167929
334 825390493570323
367 813710517305995
84 822939733715768
32 605102311580436
74 125941938756840
24 88420042986267
153 607423293538264
172 983202426465334
116 475140435636488
313 30304126269113
191 231063453440285
286 716041798847100
232 668305862715574
159 1996516881318
305 512433821508516
314 318055875811379
28 921362886287689
268 366376551745066
346 257063404935092
177 248486905215831
368 240119692468914
190 116420906301832
124 7575561221819
247 163176771097410
313 928281880684529
392 271161168267928
331 59764613860116
