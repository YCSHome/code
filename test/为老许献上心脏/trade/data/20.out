8944261064355974
